src/components/icons/ErrorBoundaryIcon.tsx

```diff
@@ -0,0 +1,46 @@
+const ErrorIcon = () => (
```

Comment:
We weren't really supposed to use functional components at this stage, but oh well